### ISSUE TITLE
Add Version Display

### ISSUE THREAD

**Issue by 0xsingular** [teaxyz/teaBASE#19]
Description:
It would be great if teaBASE showed the current version somewhere in the app. This would make it easier to know what version is being used.

Benefit:
Helps users quickly verify their version for updates.

**Comment by stevenlei** [comment]
Yes the version number is displayed at the bottom, maybe the text color is too light?

<img width="708" alt="image" src="https://github.com/user-attachments/assets/65eadc47-c393-4538-bb5b-27b24c783371" />

**Comment by 0xsingular** [comment]
Yes, some contrast is needed because it’s basically the same color as the background
